static/js/reducers/index.js

```diff
@@ -28,38 +28,42 @@ import {
   FETCH_SUCCESS,
 } from '../actions';
 
+import { HIGH_SCHOOL, ASSOCIATE, BACHELORS, MASTERS, DOCTORATE } from '../constants';
+
 export const INITIAL_DIALOG_STATE = {
   openDialog: false,
   degreeLevel: '',
-  educationIndex: 0
+  educationIndex: null,
+  educationId: null
 };
 export const educationDialog = (state = INITIAL_DIALOG_STATE, action) => {
   switch (action.type) {
     case SHOW_EDUCATION_FORM_DIALOG:
       return Object.assign({}, state, {
         openDialog: true,
         degreeLevel: action.payload.level,
-        educationIndex: action.payload.index
+        educationIndex: action.payload.index,
+        educationId: action.payload.education_id
       });
     case HIDE_EDUCATION_FORM_DIALOG:
       return INITIAL_DIALOG_STATE;
     default:
       return state;
   }
 };
-export const INITIAL_EDUCATION_LEVEL_STATE = {
-  hs: true,
-  a: true,
-  b: true,
-  m: false,
-  d: false
-};
+let education_level_state = {};
+education_level_state[HIGH_SCHOOL] = true;
+education_level_state[ASSOCIATE] = true;
+education_level_state[BACHELORS] = true;
+education_level_state[MASTERS] = false;
+education_level_state[DOCTORATE] = false;
+
+export const INITIAL_EDUCATION_LEVEL_STATE = education_level_state;
+
 export const educationLevels = (state = INITIAL_EDUCATION_LEVEL_STATE, action) => {
   switch (action.type) {
     case TOGGLE_EDUCATION_LEVEL:
-      let updated_state = {};
-      updated_state[action.payload.level]= !state[action.payload.level];
-      return Object.assign({}, state, updated_state);
+      return Object.assign({}, state, action.payload.new_state);
```

Comment:
Can you use camelcase here? Unless we are dealing with data from the REST API we should use camelcase everywhere